samples/httpbin/sample-client/fortio-deploy.yaml

```diff
@@ -6,6 +6,14 @@ spec:
   replicas: 1
   template:
     metadata:
+      annotations:
+        # This annotation allows istio-proxy sidecar to become ready.  Without
+        # it Envoy logs "Envoy proxy is NOT ready" and "envoy missing listener for inbound application port"
+        status.sidecar.istio.io/port: "0"
```

Comment:
You should document better what's going on here. Disabling readiness? Why is this needed for Fortio, but not other clients, like the sleep sample?
^^^ @ldemailly

Reply:
@nmittler I did `port:0` because you recommended it on 
https://github.com/istio/istio/issues/9504#issuecomment-439432130 .  I don't understand why it is needed.  Can you explain it to Frank and I?

Reply:
@frankbu The sleep sample doesn't need this because it comes with a Service even though you might think it is headless.  _samples/sleep/sleep.yaml_.  I also wonder why Sleep is considered 2/2 Ready because it's Service is on port 80 but `netstat -tulpn` and `curl localhost:80` show it is not listening on that port.

Reply:
Assuming this is a more general problem, I think we should add an entry in the operations guide to explain this issue. It can point to this Fortio deployment in the httpbin sample as an example. 

Maybe something about when you need to use the port 0 annotation in here: https://preliminary.istio.io/help/ops/traffic-management/deploy-guidelines/

The statsInclusion annotation should be explained somewhere too. Maybe here? https://preliminary.istio.io/help/ops/telemetry/

Reply:
I will write something about the `statsInclusionPrefixes` annotation.